beetsplug/parentwork.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2017, Dorian Soergel.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""Gets work title, disambiguation, parent work and its disambiguation,
+composer, composer sort name and performers
+"""
+
+from __future__ import division, absolute_import, print_function
+
+from beets import ui
+from beets.plugins import BeetsPlugin
+
+import musicbrainzngs
+
+
+def work_father(mb_workid, work_date=None):
+    """ This function finds the id of the father work given its id"""
```

Comment:
The naming is a little unclear here—what exactly is a "father work" and how is it different from a "parent work"? Maybe something like "parent" vs. "ancestor" would be clearer? Also, it would be nice if the docstring described (in a sentence or two) exactly what's these things mean instead of just referencing the name.

Also, this function seems to return two things, not just one.

Reply:
Also, FWIW, it's not necessary to say "This function does X…" It's usually more appropriate to just say "Do X." This series of examples might be helpful:
https://blog.dolphm.com/pep257-good-python-docstrings-by-example/

Reply:
Thanks for the link! Yes, in the old PR I got requested to fetch the composition date, so I fetch it at each level until I find a date. It's not very satisfactory, I feel like this plugin is not really the place to do this, but it's possible.

Reply:
I don't like using 'ancestor', simply because 'ancestorwork' looks bad, I prefer 'parentwork'. If I clarify it in the docs, would it be fine?

Reply:
Ok I clarified the docstring, this should be ready to merge now.